### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by L00163105**
Update README teams Social Contract section

### CODE REVIEW

**Review by JoseSunoj — CHANGES_REQUESTED** [review]
line 128, Autoatedbot (Automated bot), line 150 TDB(TBD), line 183 spring(sprint). We can add internal links to the table of contents to avoid scrolling. 'For More Information' can be a sub-heading with contents as a table. One item in the table of contents named 'Considerations' is not having space in the file; either we can exclude it from the contents or add a section inside the file or ask Ruth about it. We can add bullets to the lists in 'Useful links'. May need a tab right to nicely align the team members.

**Review by Robert-Tokarz — APPROVED** [review]
Thanks for adding me to Rockstar.

**Review by damogallagher — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -126,16 +127,16 @@ These are the current Risks on the project, re-aligned on a weekly basis
 
 ### Meetings
 
-    Stand-ups will occur on Every Wednesday during class and Fridays at 8pm GMT+1.
+    Stand-ups will occur on Every Wednesday during class and Fridays at 8pm GMT+1 using Autoatedbot.
```
Comment (sharonO13) [inline README.md:130]: Correction needed for Automated bot

**Review by sharonO13 — COMMENTED** [review]
Updates look good, just a spell check on the automated standup for Automated Bot

**Review by JoseSunoj — CHANGES_REQUESTED** [review]
Since we decided to move Friday's stand-up to Monday, we have to note the changes.

**Comment by L00163105** [comment]
> Since we decided to move Friday's stand-up to Monday, we have to note the changes.

Changed Friday references to Monday

**Review by JoseSunoj — CHANGES_REQUESTED** [review]
Please consider the following:
1. We can add internal links to the table of contents to avoid scrolling. 
2. 'For More Information' can be a sub-heading with contents as a table. 3. 
3. One item in the table of contents named 'Considerations' is not having space in the file (missing in the original file); either we can exclude it from the contents or add a section inside the file or ask Ruth about it. We can add bullets to the lists in 'Useful links'. 
4. May need a tab right to nicely align the team members.

**Comment by L00163105** [comment]
> Please consider the following:
> 
> 1. We can add internal links to the table of contents to avoid scrolling. / done
> 2. 'For More Information' can be a sub-heading with contents as a table. / done
> 3. One item in the table of contents named 'Considerations' is not having space in the file (missing in the original file); either we can exclude it from the contents or add a section inside the file or ask Ruth about it.  / not sure about this
> 4. We can add bullets to the lists in 'Useful links'. /done 
> 5. May need a tab right to nicely align the team members. /done

**Review by JoseSunoj — APPROVED** [review]

**Pull request merged**